stdlib/runtime/malloc.gr

```diff
@@ -0,0 +1,275 @@
+/* compilation-mode: malloc */
+
+import WasmI32, { 
```

Comment:
Maybe, but he's only second-degree familiar with that file (since it's been rewritten since the original implementation). I think the best way to review this file is to review the original C code and compare the two. You can find it on page 185 of this book (pdf page 199): https://kremlin.cc/k&r.pdf

Though obviously, would love for him to take a look if he can.

Reply:
🙈 Don't ask me to look at C